packages/site/src/utils/snap.ts

```diff
@@ -53,13 +53,64 @@ export const getSnap = async (version?: string): Promise<Snap | undefined> => {
 };
 
 /**
- * Invoke the "hello" method from the example snap.
+ * Get the public key from the installed snap.
  */
 
-export const sendHello = async () => {
-  await window.ethereum.request({
+export const getPublicKey = async () => {
+  return await window.ethereum.request({
     method: 'wallet_invokeSnap',
-    params: { snapId: defaultSnapOrigin, request: { method: 'hello' } },
+    params: {
+      snapId: defaultSnapOrigin,
+      request: {
+        method: 'getPublicKey',
+        params: {
+          path: ['m', "44'", "1'"],
```

Comment:
I don't recall the details of BIP-32 very well - can we document the meaning of this magic string and link to bip32. Also, let's consider pulling the `path` out as a constant since it's repeated in a few places

Reply:
This is a tricky item indeed. I added a readme comment here: https://github.com/Sovereign-Labs/sov-snap/blob/1cc2f3a1f2b72cedfacc53b2e5065e57b31ed8fc/packages/snap/README.md?plain=1#L5

The problem is that Metamask demands we specify the coin type on the manifest file (hence, one coin type per snap). It means we will have to generate a new snap if we want to use a different coin type. I set this one to `testnet`, but we have an extensive [list](https://github.com/satoshilabs/slips/blob/master/slip-0044.md#registered-coin-types)